airflow/www/static/js/grid/renderTaskRows.jsx

```diff
@@ -88,11 +87,12 @@ const TaskInstances = ({
   </Flex>
 );
 
+const storageKey = `${dagId}-open-groups`;
```

Comment:
```suggestion
const storageKey = `${dagId}/open-groups`;
```

Just to avoid some kind of possible clash in future? (We could also use `:` instead -- a character that's not allowed in a dag_id)

Reply:
Good idea. Also, updated the variable name